NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -396,6 +438,27 @@ public class NightscoutUploader {
         }
     }
     
+    public func uploadEntries(_ entries: [NightscoutEntry], completion: @escaping (Result<Bool, Error>) -> Void) {
```

Comment:
[nit] There seems to be redundant code in these new functions. Anyway to refactor?

Reply:
@nhamming Would you please review the latest that includes changes for the above?